featurebyte/__init__.py

```diff
@@ -1,10 +1,12 @@
 """Python Library for FeatureOps"""
 
-from typing import Any, List, Optional
+# Ignore import errors
+# ruff: noqa: F401
 
```

Comment:
We have imports that are not used within the __init__.py. 
If we do not ignore F401, it will cause those import to be added to __all__